src/Smp.Web/ClientApp/src/app/messages/messages.component.ts

```diff
@@ -7,9 +16,116 @@ import { Component, OnInit } from '@angular/core';
 })
 export class MessagesComponent implements OnInit {
 
-  constructor() { }
+  @ViewChild(ConversationComponent)
+  conversation: ConversationComponent;
+
+  public conversations: ExtendedConversation[];
+
+  constructor(private globalHelper: GlobalHelper,
+    private conversationsService: ConversationsService,
+    private messagesService: MessagesService,
+    private usersService: UsersService) { }
 
   ngOnInit() {
+    this.fetchConversationsData();
+  }
+
+  public getConversationPicture(conversationId: string): string {
```

Comment:
I imagine these methods could be refactored to save a few lines, but it seems like something we can just revisit in the future.